extensions/opentelemetry/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/GrpcOpenTelemetryTest.java

```diff
@@ -237,10 +239,57 @@ void stream() {
         assertEquals(server.getTraceId(), client.getTraceId());
     }
 
+    @Test
+    void streamingBlocking() {
+        Multi<Item> request = Multi.createFrom().items(item("Goku"), item("Vegeta"), item("Piccolo"), item("Beerus"),
+                item("Whis"));
+        Multi<Item> response = streamingStub.pipeBlocking(request);
```

Comment:
This is basically the same as previous streaming() test, not sure what you are resting here...

Reply:
It uses the `pipeBlocking` method, which has a `@Blocking` annotation. This will force gRPC to block, and the blocking code will be executed in another thread, so we lose the context. We need to also propagate the `Scope` with the gRPC invocation to ensure you are in the right OTel Context.

Reply:
got it, thanks!